pkg/status/status.go

```diff
@@ -359,6 +359,7 @@ func (s *status) monitorClusterStatus() {
 			if s.clusterOperator == nil {
 				conditionListBuilder := clusterStatusListBuilder()
 				conditionListBuilder(configv1.OperatorProgressing, configv1.ConditionTrue, fmt.Sprintf("Progressing towards release version: %s", s.version))
+				conditionListBuilder(configv1.OperatorUpgradeable, configv1.ConditionFalse, "Marketplace is not upgradeable")
```

Comment:
Why is the operator not upgradeable in this scenario? I can't think of a scenario where we are not upgradeable. Moreover if this condition is false, does CVO wait until it becomes true to attempt an upgrade?

Reply:
If the Operator is progressing to a new version while performing a migration we would not want to report that we are available for an upgrade.

Reply:
That should be reflected in #207 and not here.

Reply:
That's fair, I will update the PR.